frontend/packages/dev-console/integration-tests/features/Knative/filters-Serving-Eventing-Admin.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Filters on Serving and Eventing page
+    As a user, I should be able to use filters as per the use
+
+
+    Background:
+        Given user is at administrator perspective
+        And user has installed OpenShift Serverless Operator
+
+
+    @regression
+    Scenario: Search by name correctly
+        Given user is at Serving page
+        And user has created knative service "hello-openshift"
+        And user has selected Services tab
+        When user clicks on dropdown button
+        And user selects Name
+        And user enters "hello-openshift"
+        Then user will see KSVC by name "hello-openshift"
+        And user will see Clear all filters
+
+
+    @regression
+    Scenario: Search by label correctly
+        Given user is at Serving page
+        And user has created knative service "hello-openshift"
+        And user has selected Routes tab
+        When user clicks on dropdown button
+        And user selects Label
+        And user enters "hello-openshift"
+        Then user will see routes for KSVC by name "hello-openshift"
+        And user will see Clear all filters
+        
+        
+    @regression
+    Scenario: Search by name incorrectly
+        Given user is at Serving page
+        And user has created knative service "hello-openshift"
+        And user has selected Revisions tab
+        When user clicks on dropdown button
+        And user selects Name
+        And user enters "xyz"
+        Then user will see message "No Revisions found"
+        And user will see Clear all filters
+
+
+    @regression
+    Scenario: Event Sources Filter
+        Given user has created knative service "hello-openshift"
+        And user has created ApiServer Source
+        And user has created Ping Source
+        And user has created Sink Binding
+        And user has created Container Source
+        And user is at Event Sources tab
+        When user clicks on Filter dropdown
+        Then user will see digit "1" in front of ApiServer, Ping, Sink, Container Sources checkboxes
+        And user will see only ApiServer source when checked checkbox in fornt of ApiServer Source
```

Comment:
```suggestion
        And user will see only ApiServer source when checked the checkbox in front of ApiServer Source
```

Reply:
These are the actions. Do yu think they are supposed to be in `Then`

Reply:
Updated

Reply:
I thought of it. But then I need to create different scenarios for every checkbox present in that filter drop down which is unnecessary. That's why I arranged it in such a way.